TheWarWithin/ShamanRestoration.lua

```diff
@@ -640,6 +649,7 @@ spec:RegisterAbilities( {
         notalent = "surging_totem",
 
         handler = function ()
+            setCooldown( "downpour", 0 )
```

Comment:
Does `healing_rain`'s cooldown change based on Surging Totem as well?  (Like `downpour`?)

Reply:
Also, does this CD reset require any particular talent selections?  Might save a few CPU cycles if we make it conditional:
`if talent.downpour.enabled then` ...

Reply:
downpour doesn't even really have a real CD I don't think. It's weird, you basically get 1 limited time cast of it when you use a healing rain or surging totem. This is basically just to prevent any weird cases where it might try and ask you to cast it again. I can remove if you think it's unnecessary.

Reply:
Healing rain sort of becomes surging totem, with longer duration and cooldown. It gets replaced. Will make change to healing rain buff duration to reflect that.